README.md

```diff
@@ -1 +1,32 @@
-# spitoruler
\ No newline at end of file
+# SPITO
+SPTIO (Simple Powerful Interprocess Tool to Overpower) has been designed
+to allow users easily swap and edit configuration files.
+
+## Use cases
+**For a while it's a myth language**
+
+### Simple config
+- used rule: samba-guest-are-god
+- purpose: use whole ready-to-use samba config
+```shell
+spito -r samba-guests-are-god
+```
+
+### Variable config
+- used rule: wireguard
+- purpose: use config that may need additional information
+- good to notice: rule will generate keys for you
+
+```shell
+spito wireguard -o NUMBER_OF_CLIENT=3
```

Comment:
Same as above, we cannot directly specify configuration variable

Reply:
All configuration options should be in the rule itself

Reply:
> All configuration options should be in the rule itself
No they shouldn't. Some options you want to define yourself.